packages/aws-cdk-lib/aws-events-targets/lib/ecs-task.ts

```diff
@@ -81,6 +95,20 @@ export interface EcsTaskProps extends TargetBaseProps {
    * @default - ECS will set the Fargate platform version to 'LATEST'
    */
   readonly platformVersion?: ecs.FargatePlatformVersion;
+
+  /**
+    * Specifies whether to propagate the tags from the task definition to the task. If no value is specified, the tags are not propagated.
+    *
+    * @default - Tags will not be propagated
+    */
+  readonly propagateTags?: ecs.PropagatedTagSource
+
+  /**
+     * The metadata that you apply to the task to help you categorize and organize them. Each tag consists of a key and an optional value, both of which you define.
+     *
+     * @default - No tags are applied to the task
+     */
+  readonly tagList?: Tag[]
```

Comment:
What about just `tags`? I don't think we need to say `List` since the type tells
you it is a list.

Reply:
I agree that `tags` is probably better here. I chose `tagList` initially to match the CloudFormation parameter name https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-ecsparameters.html#cfn-events-rule-ecsparameters-taglist. Since this is a level 2 construct though I suppose there's no need to faithfully match that naming.